core/src/main/scala/quasar/plugin/postgres/package.scala

```diff
@@ -78,6 +89,57 @@ package object postgres {
   def hygienicIdent(ident: Ident): Ident =
     s""""${ident.replace("\"", "\"\"")}""""
 
+  def specifyColumnFragments[F[_]: ApplicativeError[?[_], Throwable]](
+    cols: NonEmptyList[Column[ColumnType.Scalar]])
+      : F[NonEmptyList[Fragment]] =
+    cols.traverse(columnSpec).fold(
+      invalid => ApplicativeError[F, Throwable].raiseError(new ColumnTypesNotSupported(invalid)),
+      _.pure[F])
+
+  def specifyColumnFragment[F[_]: ApplicativeError[?[_], Throwable]](
+    col: Column[ColumnType.Scalar])
+      : F[Fragment] =
+    columnSpec(col).fold(
+      invalid => ApplicativeError[F, Throwable].raiseError(new ColumnTypesNotSupported(invalid)),
+      _.pure[F])
+
+  def createTable(log: Logger)(table: Table, colSpecs: NonEmptyList[Fragment])
+      : ConnectionIO[Int] = {
+    val preamble =
+      fr"CREATE TABLE IF NOT EXISTS" ++ Fragment.const(hygienicIdent(table))
```

Comment:
I think this should be `CREATE TABLE`, otherwise it is identical to `createTableIfNotExists`.

Reply:
Fixed!